static/js/services/languages.js

```diff
@@ -0,0 +1,20 @@
+var _ = require('underscore');
+
+/*
+ * Get all enabled languages
```

Comment:
The style I've been going for is one service per file because it makes them easier to find IMO. I was tempted to split up language.js but didn't want to conflate this PR.